snapcraft/tests/fake_servers.py

```diff
@@ -841,6 +841,10 @@ def do_GET(self):  # noqa: C901
             self._DEV_API_PATH, 'snaps/no-dev/developers')
         bad_developers_path = urllib.parse.urljoin(
             self._DEV_API_PATH, 'snaps/badrequest/developers')
+        revoked_developers_path = urllib.parse.urljoin(
+            self._DEV_API_PATH, 'snaps/revoked/developers')
+        not_revoked_developers_path = urllib.parse.urljoin(
+            self._DEV_API_PATH, 'snaps/no-revoked/developers')
```

Comment:
Note the refactors here to make this shorter:
https://github.com/snapcore/snapcraft/pull/1288/files#diff-2a6de018c4c141826eaa1f76b9b4c31aR839

They will cause conflicts with your branch.